src/Common/tests/System.Net/Capability.cs

```diff
@@ -1,145 +0,0 @@
-﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Np: I can add this back. For the future, please leave files in place until we've finished all work.
For the merge with corefx, we can pick only the files that are required by build for the PR. That means that this file shouldn't be picked.

In parallel, we're working on this capability system for System.Net.Http. I think this will end up split in multiple partial classes similar to Capability.Sockets.cs.